src/components/AddressSearch/index.jsx

```diff
@@ -1,25 +1,21 @@
 import React from 'react';
 import { Spinner } from '@patternfly/react-core/dist/js/experimental';
 import api from 'api';
-import ParcelMap from 'components/ParcelMap';
 import SearchForm from './components/SearchForm';
 import SearchResultsList from './components/SearchResultsList';
-import styles from './index.module.css';
+import styles from './index.module.scss';
 
 /** @typedef {import('api').Parcel} Parcel */
 
 /**
- * MapSearch component props
+ * AddressSearch component props
  * @typedef {Object} Props
- * @property {function(Parcel): void} onSelectParcel - Callback to handle when the user
```

Comment:
The `onSelectParcel` prop was not removed. It's still referenced inside `render()`.
Also, we should add the `onHasSearchResults` prop in here. It would look something like this:

`@property {function(boolean): void} onHasSearchResults`

Reply:
Sorry, deprecated removal from a previous iteration. Editing & adding to a cleanup commit!